data/saas/dataset/alchemer_dataset.yml

```diff
@@ -0,0 +1,9 @@
+dataset:
+  - fides_key: <instance_fides_key>
+    name: alchemer_erasure_only
+    description: >-
+      A sample dataset representing the Alchemer erasure only connector for
+      Fides
```

Comment:
```suggestion
    description: A sample dataset representing the Alchemer erasure only connector for Fides
```

Reply:
Removed the >- but I can't get the text of the description to stay on that line - when I save it drops it down =(